File: IDAUtil/Model/ZV04HNProperty.vb
Imports System.ComponentModel.DataAnnotations.Schema

Public Class ZV04HNProperty
    <Column("[status]")>
    Public Property status As Long
    <Column("[order]")>
    Public Property order As Integer
    <Column("[delivery]")>
    Public Property delivery As Long
    <Column("[shipment]")>
    Public Property shipment As String
    <Column("[docTyp]")>
    Public Property docTyp As String
    <Column("[docDate]")>
    Public Property docDate As Date
    <Column("[delBlock]")>
    Public Property delBlock As String
    <Column("[delBlockDesc]")>
    Public Property delBlockDesc As String
    <Column("[ordNetValue]")>
    Public Property ordNetValue As Double
    <Column("[reqDelDate]")>
    Public Property reqDelDate As Date
    <Column("[pricingDate]")>
    Public Property pricingDate As String
    <Column("[pODate]")>
    Public Property pODate As String
    <Column("[loadingDate]")>
    Public Property loadingDate As Date
    <Column("[plant]")>
    Public Property plant As String
    <Column("[orderQty]")>
    Public Property orderQty As Double
    <Column("[confirmedQty]")>
    Public Property confirmedQty As Double
    <Column("[deliveryQty]")>
    Public Property deliveryQty As Double
    <Column("[deliveryNetValue]")>
    Public Property deliveryNetValue As Double
    <Column("[FillRate]")>
    Public Property FillRate As Double
    <Column("[pONumber]")>
    Public Property pONumber As String
    <Column("[soldto]")>
    Public Property soldto As Long
    <Column("[soldtoName]")>
    Public Property soldtoName As String
    <Column("[salesOffice]")>
    Public Property salesOffice As String
    <Column("[salesGroup]")>
    Public Property salesGroup As String
    <Column("[salesDistrict]")>
    Public Property salesDistrict As String
    <Column("[backorderIndicator]")>
    Public Property backorderIndicator As String
    <Column("[shipto]")>
    Public Property shipto As long
    <Column("[shiptoName]")>
    Public Property shiptoName As String
    <Column("[cSRASR]")>
    Public Property cSRASR As String
    <Column("[csrasrName]")>
    Public Property csrasrName As String
    <Column("[scjAgent]")>
    Public Property scjAgent As String
    <Column("[scjAgentName]")>
    Public Property scjAgentName As String
    <Column("[scjAgentReg]")>
    Public Property scjAgentReg As String
    <Column("[scjAgentRegName]")>
    Public Property scjAgentRegName As String
    <Column("[orderTlfp]")>
    Public Property orderTlfp As Double
    <Column("[orderPlt]")>
    Public Property orderPlt As Double
    <Column("[orderLayers]")>
    Public Property orderLayers As Double
    <Column("[orderLooseCs]")>
    Public Property orderLooseCs As Double
    <Column("[orderTotCs]")>
    Public Property orderTotCs As Double
    <Column("[tlfpCmmt]")>
    Public Property tlfpCmmt As Double
    <Column("[pltCmmt]")>
    Public Property pltCmmt As Double
    <Column("[layersCmmt]")>
    Public Property layersCmmt As Double
    <Column("[looseCsCmmt]")>
    Public Property looseCsCmmt As Double
    <Column("[totCsCmmt]")>
    Public Property totCsCmmt As Double
    <Column("[headerWeight]")>
    Public Property headerWeight As Double
    <Column("[wtUnit]")>
    Public Property wtUnit As String
    <Column("[headerCube]")>
    Public Property headerCube As Double
    <Column("[volUnit]")>
    Public Property volUnit As String
    <Column("[ediSummaryData]")>
    Public Property ediSummaryData As String
    <Column("[zv04Report]")>
    Public Property zv04Report As String
    <Column("[ordDesc]")>
    Public Property ordDesc As String
    <Column("[description]")>
    Public Property description As String
    <Column("[createdBy]")>
    Public Property createdBy As String
    <Column("[route]")>
    Public Property route As String
    <Column("[shipToPostalCode]")>
    Public Property shipToPostalCode As String
    <Column("[shipToRegion]")>
    Public Property shipToRegion As String
    <Column("[shipToRegionName]")>
    Public Property shipToRegionName As String
    <Column("[shipToCity]")>
    Public Property shipToCity As String
    <Column("[shipToStreet]")>
    Public Property shipToStreet As String
    <Column("[soldToPostalCode]")>
    Public Property soldToPostalCode As String
    <Column("[soldToRegion]")>
    Public Property soldToRegion As String
    <Column("[soldToRegionName]")>
    Public Property soldToRegionName As String
    <Column("[soldToCity]")>
    Public Property soldToCity As String
    <Column("[soldToStreet]")>
    Public Property soldToStreet As String
    <Column("[realOpenOrderQty]")>
    Public Property realOpenOrderQty As String
    <Column("[realOpenOrderNetValue]")>
    Public Property realOpenOrderNetValue As Double
    <Column("[appointmentTimes]")>
    Public Property appointmentTimes As String
    <Column("[csrNotes]")>
    Public Property csrNotes As String
    <Column("[ediTransmittedText]")>
    Public Property ediTransmittedText As String
    <Column("[deliveryInstructions]")>
    Public Property deliveryInstructions As String
    <Column("[deliveryNote2]")>
    Public Property deliveryNote2 As String
    <Column("[carrierInstructions]")>
    Public Property carrierInstructions As String
End Class

File: IDAUtil/Model/Properties/TcodeProperty/ZV04Obj/ZV04HNProperty.cs
﻿using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace IDAUtil.Model.Properties.TcodeProperty.ZV04Obj {
    public class ZV04HNProperty {
        [Column("[status]")] public string status { get; set; }
        [Column("[order]")] public int order { get; set; }
        [Column("[delivery]")] public int delivery { get; set; }
        [Column("[shipment]")] public string shipment { get; set; }
        [Column("[docTyp]")] public string docTyp { get; set; }
        [Column("[docDate]")] public DateTime docDate { get; set; }
        [Column("[delBlock]")] public string delBlock { get; set; }
        [Column("[delBlockDesc]")] public string delBlockDesc { get; set; }
        [Column("[ordNetValue]")] public double ordNetValue { get; set; }
        [Column("[reqDelDate]")] public DateTime reqDelDate { get; set; }
        [Column("[pricingDate]")] public string pricingDate { get; set; }
        [Column("[pODate]")] public string pODate { get; set; }
        [Column("[loadingDate]")] public DateTime loadingDate { get; set; }
        [Column("[plant]")] public string plant { get; set; }
        [Column("[orderQty]")] public double orderQty { get; set; }
        [Column("[confirmedQty]")] public double confirmedQty { get; set; }
        [Column("[deliveryQty]")] public double deliveryQty { get; set; }
        [Column("[deliveryNetValue]")] public double deliveryNetValue { get; set; }
        [Column("[FillRate]")] public double FillRate { get; set; }
        [Column("[pONumber]")] public string pONumber { get; set; }
        [Column("[soldto]")] public int soldto { get; set; }
        [Column("[soldtoName]")] public string soldtoName { get; set; }
        [Column("[salesOffice]")] public string salesOffice { get; set; }
        [Column("[salesGroup]")] public string salesGroup { get; set; }
        [Column("[salesDistrict]")] public string salesDistrict { get; set; }
        [Column("[backorderIndicator]")] public string backorderIndicator { get; set; }
        [Column("[shipto]")] public int shipto { get; set; }
        [Column("[shiptoName]")] public string shiptoName { get; set; }
        [Column("[cSRASR]")] public string cSRASR { get; set; }
        [Column("[csrasrName]")] public string csrasrName { get; set; }
        [Column("[scjAgent]")] public string scjAgent { get; set; }
        [Column("[scjAgentName]")] public string scjAgentName { get; set; }
        [Column("[scjAgentReg]")] public string scjAgentReg { get; set; }
        [Column("[scjAgentRegName]")] public string scjAgentRegName { get; set; }
        [Column("[orderTlfp]")] public double orderTlfp { get; set; }
        [Column("[orderPlt]")] public double orderPlt { get; set; }
        [Column("[orderLayers]")] public double orderLayers { get; set; }
        [Column("[orderLooseCs]")] public double orderLooseCs { get; set; }
        [Column("[orderTotCs]")] public double orderTotCs { get; set; }
        [Column("[tlfpCmmt]")] public double tlfpCmmt { get; set; }
        [Column("[pltCmmt]")] public double pltCmmt { get; set; }
        [Column("[layersCmmt]")] public double layersCmmt { get; set; }
        [Column("[looseCsCmmt]")] public double looseCsCmmt { get; set; }
        [Column("[totCsCmmt]")] public double totCsCmmt { get; set; }
        [Column("[headerWeight]")] public double headerWeight { get; set; }
        [Column("[wtUnit]")] public string wtUnit { get; set; }
        [Column("[headerCube]")] public double headerCube { get; set; }
        [Column("[volUnit]")] public string volUnit { get; set; }
        [Column("[ediSummaryData]")] public string ediSummaryData { get; set; }
        [Column("[zv04Report]")] public string zv04Report { get; set; }
        [Column("[ordDesc]")] public string ordDesc { get; set; }
        [Column("[description]")] public string description { get; set; }
        [Column("[createdBy]")] public string createdBy { get; set; }
        [Column("[route]")] public string route { get; set; }
        [Column("[shipToPostalCode]")] public string shipToPostalCode { get; set; }
        [Column("[shipToRegion]")] public string shipToRegion { get; set; }
        [Column("[shipToRegionName]")] public string shipToRegionName { get; set; }
        [Column("[shipToCity]")] public string shipToCity { get; set; }
        [Column("[shipToStreet]")] public string shipToStreet { get; set; }
        [Column("[soldToPostalCode]")] public string soldToPostalCode { get; set; }
        [Column("[soldToRegion]")] public string soldToRegion { get; set; }
        [Column("[soldToRegionName]")] public string soldToRegionName { get; set; }
        [Column("[soldToCity]")] public string soldToCity { get; set; }
        [Column("[soldToStreet]")] public string soldToStreet { get; set; }
        [Column("[realOpenOrderQty]")] public string realOpenOrderQty { get; set; }
        [Column("[realOpenOrderNetValue]")] public double realOpenOrderNetValue { get; set; }
        [Column("[appointmentTimes]")] public string appointmentTimes { get; set; }
        [Column("[csrNotes]")] public string csrNotes { get; set; }
        [Column("[ediTransmittedText]")] public string ediTransmittedText { get; set; }
        [Column("[deliveryInstructions]")] public string deliveryInstructions { get; set; }
        [Column("[deliveryNote2]")] public string deliveryNote2 { get; set; }
        [Column("[carrierInstructions]")] public string carrierInstructions { get; set; }
    }
}